src/platform_impl/web/web_sys/pointer.rs

```diff
@@ -173,34 +198,48 @@ impl PointerHandler {
                     fn has_get_coalesced_events(this: &PointerEventExt) -> JsValue;
                 }
 
-                match event.pointer_type().as_str() {
-                    "touch" => {
-                        if prevent_default {
-                            // prevent scroll on mobile web
-                            event.prevent_default();
-                        }
```

Comment:
This was added by https://github.com/rust-windowing/winit/pull/2216, it doesn't seem to do anything anymore. @madsmtm do you still remember why this was added specifically?

Reply:
Nope, don't remember, I didn't (and don't) know enough about the web to question it then. Maybe @rukai knows why/if it was necessary?

Reply:
I believe this was a necessary part of https://github.com/rust-windowing/winit/pull/2216#discussion_r962224122

As long as the functionality added in the pr is upheld, then I have no issue with it being removed 
That is, when winit is configured to, scrolling should be disabled on both web and mobile browsers.

Reply:
Yeah, from what I tried the functionality is upheld.
Thanks!